use reqwest::Error;

#[tokio::main]
async fn main() -> Result<(), Error>{
    Ok(())
}
